plugins/search-react/src/alpha.tsx

```diff
@@ -87,6 +87,13 @@ export type SearchResultItemExtensionOptions<
 export function createSearchResultListItemExtension<
   TConfig extends { noTrack?: boolean },
 >(options: SearchResultItemExtensionOptions<TConfig>) {
+  const id = `plugin.search.result.item.${options.id}`;
+
+  const attachTo = options.attachTo ?? {
+    id: 'plugin.search.page',
+    input: 'items',
+  };
```

Comment:
This one too, why not keep it where it is? 😁

Reply:
I think I did in the search create result item extension following the config schema pattern, but I reverted it in all places 🙂